<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VCCueListProperties</class>
 <widget class="QDialog" name="VCCueListProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cue list properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="m_nameLabel">
     <property name="text">
      <string>Cue list name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="m_nameEdit">
     <property name="toolTip">
      <string>The name of the cue list widget</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="m_tab">
     <property name="toolTip">
      <string/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Cues">
      <attribute name="title">
       <string>Cues</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" rowspan="5">
        <widget class="QTreeWidget" name="m_list">
         <property name="toolTip">
          <string>List of scenes that can be activated with this cue list</string>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Number</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Scene</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="m_addButton">
         <property name="toolTip">
          <string>Add functions to the cue list</string>
         </property>
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/edit_add.png</normaloff>:/edit_add.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="m_removeButton">
         <property name="toolTip">
          <string>Remove selected functions from the cue list</string>
         </property>
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/edit_remove.png</normaloff>:/edit_remove.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="m_raiseButton">
         <property name="toolTip">
          <string>Raise the selected function</string>
         </property>
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/up.png</normaloff>:/up.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QToolButton" name="m_lowerButton">
         <property name="toolTip">
          <string>Lower the selected function</string>
         </property>
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/down.png</normaloff>:/down.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="NextCue">
      <attribute name="title">
       <string>Next Cue</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Key Combination</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="m_nextKeyEdit">
            <property name="toolTip">
             <string>The key combination used to step to the next cue</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_nextAttachButton">
            <property name="toolTip">
             <string>Bind a key combination to skip to the next cue</string>
            </property>
            <property name="text">
             <string notr="true">...</string>
            </property>
            <property name="icon">
             <iconset resource="main.qrc">
              <normaloff>:/key_bindings.png</normaloff>:/key_bindings.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_nextDetachButton">
            <property name="toolTip">
             <string>Clear the key binding</string>
            </property>
            <property name="text">
             <string notr="true">...</string>
            </property>
            <property name="icon">
             <iconset resource="main.qrc">
              <normaloff>:/fileclose.png</normaloff>:/fileclose.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>External Input</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Input universe</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="m_nextInputUniverseEdit">
            <property name="toolTip">
             <string>Input universe for skipping to the next cue</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Input channel</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="m_nextInputChannelEdit">
            <property name="toolTip">
             <string>Input channel for skipping to the next cue</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>225</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="m_nextAutoDetectInputButton">
            <property name="toolTip">
             <string>When toggled, you can click an external button to assign it as the control that skips to the next cue.</string>
            </property>
            <property name="text">
             <string>Auto Detect</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="m_nextChooseInputButton">
            <property name="toolTip">
             <string>Choose an input universe/channel for skipping to the next cue</string>
            </property>
            <property name="text">
             <string>Choose...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>68</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PreviousCue">
      <attribute name="title">
       <string>Previous Cue</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Key Combination</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="m_previousKeyEdit">
            <property name="toolTip">
             <string>The key combination used to step to the previous cue</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_previousAttachButton">
            <property name="toolTip">
             <string>Bind a key combination to skip to the previous cue</string>
            </property>
            <property name="text">
             <string notr="true">...</string>
            </property>
            <property name="icon">
             <iconset resource="main.qrc">
              <normaloff>:/key_bindings.png</normaloff>:/key_bindings.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_previousDetachButton">
            <property name="toolTip">
             <string>Clear the key binding</string>
            </property>
            <property name="text">
             <string notr="true">...</string>
            </property>
            <property name="icon">
             <iconset resource="main.qrc">
              <normaloff>:/fileclose.png</normaloff>:/fileclose.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>External Input</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Input universe</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="m_previousInputUniverseEdit">
            <property name="toolTip">
             <string>Input universe for skipping to the previous cue</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Input channel</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="m_previousInputChannelEdit">
            <property name="toolTip">
             <string>Input channel for skipping to the previous cue</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>225</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="m_previousAutoDetectInputButton">
            <property name="toolTip">
             <string>When toggled, you can click an external button to assign it as the control that skips to the previous cue.</string>
            </property>
            <property name="text">
             <string>Auto Detect</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="m_previousChooseInputButton">
            <property name="toolTip">
             <string>Choose an input universe/channel for skipping to the previous cue</string>
            </property>
            <property name="text">
             <string>Choose...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VCCueListProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VCCueListProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_tab</sender>
   <signal>currentChanged(int)</signal>
   <receiver>VCCueListProperties</receiver>
   <slot>slotTabChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotTabChanged()</slot>
 </slots>
</ui>
